### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by NikoHelle**
Latest version of all translations. No untranslated keys.

### CODE REVIEW

File: src/i18n/sv.json [hunk 1]
```diff
@@ -170,7 +170,7 @@
     "ariaSelectedOption": "{{value}} har valts",
     "ariaNoSelectedItemForLabel": "{{label}} har inte valts",
     "deleteProfile": "Radera profil",
-    "description": "Helsingforsprofilen gör det möjligt att använda dina uppgifter i alla Helsingfors webbtjänster där identifiering krävs. Samma uppgifter, enkelt användbara i olika tjänster. Behändigt, eller hur?",
+    "description": "Helsinki-profilen gör det möjligt att använda dina uppgifter i alla Helsingfors webbtjänster som kräver identifiering. Samma information, enkel att använda i många olika tjänster.",
```
Comment (akikoskinen) [inline src/i18n/sv.json]: From the Swedish translation file I can find:

- Helsingfors-profil
- Helsinki-profil
- Helsingfors profil

Would probably be good to stick to only one term.

File: src/i18n/sv.json [hunk 2]
```diff
@@ -232,18 +232,18 @@
...
-  "openInNewTabAriaLabel": "SV: Avautuu uudessa välilehdessä.",
-  "openInExternalDomainAriaLabel": "SV: Siirtyy toiseen sivustoon.",
+  "openInNewTabAriaLabel": "Öppnas i en ny flik.",
+  "openInExternalDomainAriaLabel": "Gå till en annan webbplats.",
   "accessibilityStatement": "Tillgänglighetsutlåtande",
   "cityOfHelsinki": "Helsingfors stad",
-  "pageNotFoundTitle": "SV: Sivua ei löytynyt",
-  "pageNotFoundText": "SV: Tarkista URL-osoitteen oikeinkirjoitus (tarkista isot ja pienet kirjaimet sekä välimerkit). Tai palaa edelliselle sivulle napsauttamalla selaimen Edellinen-painiketta. Voit mennä myös etusivulle alla olevasta linkistä."
+  "pageNotFoundTitle": "Vi kunde inte hitta webbplatsen.",
+  "pageNotFoundText": "Kontrollera att URL-adressen är rätt skriven (kontrollera stora och små bokstäver och skiljetecken). Eller gå tillbaka till föregående sida genom att klicka på tangenten Föregående i webbläsaren) Du kan också gå tillbaka till första sidan via länken här nedan."
```
Comment (akikoskinen) [inline src/i18n/sv.json]: There's a closing parenthesis where there should be a full stop (after the word "webbläsaren").

**Review by akikoskinen — CHANGES_REQUESTED** [review]

File: src/i18n/sv.json (hunk 1 above)
Comment (NikoHelle) [reply src/i18n/sv.json]: Good catch! 👍 "Helsingfors-profil" was chosen.

File: src/i18n/sv.json (hunk 2 above)
Comment (NikoHelle) [reply src/i18n/sv.json]: Fixed!

**Review by akikoskinen — APPROVED** [review]

**Pull request merged**